src/pages/students/[student_id]/goals/[goal_id]/create.tsx

```diff
@@ -27,6 +28,16 @@ interface BenchmarkFormEntry {
   [key: string]: string | number | "";
 }
 
+const BenchmarkStepperIcon = (stepIconProps: StepIconProps) => {
+  const { completed = false } = stepIconProps;
+
+  if (completed) {
+    return <CheckCircle />;
+  } else {
+    return <TripOriginRounded />;
+  }
```

Comment:
Since the Stepper will set `completed` appropriately whether or not the step for the current icon has been completed, we can keep track of which icon to use here.

Reply:
Yeah, that's what it looks like. The `completed` property is passed to the icon component from `StepLabel` [here](https://github.com/mui/material-ui/blob/c54a42dd8b0ec8e850bd60438e60a1a8acb3b4a4/packages/mui-material/src/StepLabel/StepLabel.js#L169). Seems it's tracked within `StepContext`. I couldn't find exactly where it's computed, but its value is definitely being set appropriately.